frontend/public/components/utils/_log-window.scss

```
@@ -24,9 +24,16 @@ $color-log-window-divider: $color-log-window-header-bg;
   padding-top: 10px;
 }
 
-.log-window__lines {
+.log-window__lines, .log-window__lines-wrap{
   padding-left: 10px;
   padding-right: 10px;
+}
+
+.log-window__lines {
   white-space: pre;
   width: 0;
 }
+
+.log-window__lines-wrap {
```

```suggestion
.log-window__lines--wrap {
```

Since really what you're doing is modifying an existing rule.  See http://getbem.com/introduction/

---

Got it.  I'll make the changes.  Thanks for the link.

---

I was being extra cautious and figured there would be "newbie" mistakes so I figured I would get those out of the way before I make an official PR to the upstream master.